test/Driver/opt-record.swift

```diff
@@ -20,26 +26,41 @@ public func bar() {
   // YAML-NEXT: Pass:            sil-inliner
   // YAML-NEXT: Name:            sil.Inlined
   // YAML-NEXT: DebugLoc:
-  // YAML-NEXT:   File:            {{.*}}opt-record.swift
-  // YAML-NEXT:   Line:            [[@LINE-6]]
-  // YAML-NEXT:   Column:          3
+  // YAML:        File:            {{.*}}opt-record.swift
+  // YAML:        Line:            [[@LINE-6]]
+  // YAML:        Column:          3
   // YAML-NEXT: Function:        'bar()'
```

Comment:
This is fine but can you explain/document here why you needed these?

Reply:
The YAMLTraits in swift had a different `flow` setting for the debug location, resulting in some test changes. I'll update the description.